### PR TITLE
Broad code cleanup

### PR DESCRIPTION
**Pull request by brews**
Includes a number of fixes.

- Refactor mutable defaults from args (still may have `lambda` defaults! :-( )
- Add missing `self` to method args.
- Remove redundant parenthesis.
- Belling cats (broad exception clauses)

### CODE REVIEW

**Comment by brews** [comment]
The CI tests in this repo have been failing for a long time.

All "code_cleanup" branches passed against impact-calculations' tests (including system tests). This is ready to merge, pending reviewer approval.

**Review by jrising — APPROVED** [review]

**Pull request merged**
